draft-ietf-quic-load-balancers.md

```diff
@@ -153,6 +153,12 @@ Fundamentally, servers generate connection IDs that encode their server ID.
 Load balancers decode the server ID from the CID in incoming packets to route
 to the correct server.
 
+{{!RFC8999}} specifies that endpoints generate their own connection IDs,
```

Comment:
When you say "their own", it's possibly unclear what you mean.  If you mean that it is the connection IDs that are put in packets that are sent to the endpoint, that's correct...  But this could be interpreted in other ways that are less good.